sdk/search/Azure.Search.Documents/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,35 @@
+# Troubleshooting Azure Search SDK Issues
+
+The `Azure.Search.Documents` package provides APIs for operations on the [Azure Cognitive Search](https://docs.microsoft.com/azure/search/search-what-is-azure-search) cloud service.
+
+See [Azure SDK diagnostics](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md) for help diagnosing various problems across all our Azure SDKs for .NET.
+
+## Table of Contents
+
+* [Identifying and Troubleshooting Issues by Response Code](#troubleshooting-issues-by-response-code)
+  * [HTTP 207 Errors](#207-multi-status)
+  * [HTTP 404 Errors](#404-not-found)
+  * [HTTP 429 Errors](#429-too-many-requests)
+* [Unexpected search query results](#unexpected-search-query-results)
+
+# Troubleshooting Issues By Response Code
+See [this page](https://docs.microsoft.com/rest/api/searchservice/http-status-codes) for the common response status codes sent by the Azure Cognitive Search service.
+
+## 207 Multi-Status
```

Comment:
It might be helpful to have some code snippets where we show how to check the response status code.

Reply:
Sounds good. I'll add samples about checking the response status code in the next round of improvements.